[2025-02-10 17:35:22,295] INFO in monitor: 指标服务器启动成功 - 端口: 9090
[2025-02-10 17:35:22,295] INFO in main: 应用启动成功
[2025-02-10 17:35:26,190] ERROR in cache: 序列化数据失败: __pydantic_private__
[2025-02-10 17:35:26,193] INFO in auth_service: 为用户 1 创建访问令牌成功
[2025-02-10 17:35:26,193] INFO in auth_service: 用户 FeyFey 登录成功
[2025-02-10 17:35:26,194] INFO in response: 请求处理完成 - 路径: /api/v1/auth/login 方法: POST 处理时间: 0.25s
[2025-02-10 17:35:50,560] INFO in main: 应用关闭
[2025-02-10 17:36:36,254] INFO in monitor: 指标服务器启动成功 - 端口: 9090
[2025-02-10 17:36:36,254] INFO in main: 应用启动成功
[2025-02-10 17:36:40,745] ERROR in cache: 序列化数据失败: __pydantic_private__
[2025-02-10 17:36:40,748] INFO in auth_service: 为用户 1 创建访问令牌成功
[2025-02-10 17:36:40,748] INFO in auth_service: 用户 FeyFey 登录成功
[2025-02-10 17:36:40,749] INFO in response: 请求处理完成 - 路径: /api/v1/auth/login 方法: POST 处理时间: 0.28s
[2025-02-10 17:36:46,441] INFO in main: 应用关闭
[2025-02-10 17:38:51,275] INFO in monitor: 指标服务器启动成功 - 端口: 9090
[2025-02-10 17:38:51,275] INFO in main: 应用启动成功
[2025-02-10 17:38:53,905] INFO in auth_service: 为用户 1 创建访问令牌成功
[2025-02-10 17:38:53,905] INFO in auth_service: 用户 FeyFey 登录成功
[2025-02-10 17:38:53,907] INFO in response: 请求处理完成 - 路径: /api/v1/auth/login 方法: POST 处理时间: 0.25s
[2025-02-10 17:39:03,290] INFO in main: 应用关闭
